postgresStore.go

```diff
@@ -16,24 +16,21 @@ const (
 	UPSERT_TASK_SESSION string = `INSERT INTO task_session (username,taskid) VALUES ($1,$2) ON CONFLICT (username) DO UPDATE SET taskid=$2`
 )
 
-type PostgresStore struct {
+type DBStore struct {
 	Db *sql.DB
 }
 
-func NewPostgresStore(conn string) (*PostgresStore, error) {
-
+func NewPostgresStore(conn string) (*DBStore, error) {
```

Comment:
Great! What do you think about adding another implementation, to make sure this interface makes sense—`NewSQLiteStore`, perhaps?